allennlp/data/dataset_readers/event2mind.py

```diff
@@ -104,14 +104,15 @@ def _read(self, file_path):
                                 )
                 # Generate instances where each token of input appears once.
                 else:
-                    # To the extent that sources are duplicated in the dataset
-                    # (which appears common), we will duplicate them here.
-                    yield self.text_to_instance(source_sequence, "none", "none", "none")
                     for xintent in xintents:
```

Comment:
Is it possible to get an empty list of `xintent`? In which case, the `source_sequence` won't be listed.

Reply:
I'm a bit confused here. I thought the point was to not duplicate the source if there are multiple entries for a given target. Or was it how it was done in `event2mind-2018.10.05`? Am I missing something?
Thanks!

Reply:
You're exactly right on both counts. The issue is that this bug was there when I made the vocab for `event2mind-2018.10.05`. When I retrained today with the "corrected" vocab I noticed different -- and not clearly better -- behavior from the model. Until someone has time to devote to determining the optimal strategy* for vocab generation I think it's best that the code at least matches the model.

*Personally I liked what you were saying about using a permissive source vocab and a restricted target vocab. That would explain what I've seen so far.